students/jaeyeon-won/project_westagram/accounts/views.py

```diff
@@ -0,0 +1,65 @@
+import json
+import re
+
+from django.views import View
+from django.http  import JsonResponse
+
+from .models      import User
+
+class SignupView(View):
+    def post(self,request):
+        data = json.loads(request.body)
+
+        try:
+            name                    = data['name']
+            phone_number            = data['phone_number']
+            email                   = data['email']
+            username                = data ['username']
+            password                = data['password']
+            email_vaildation        = re.match('[a-zA-Z0-9._+-]+@[a-z0-9-]+\.[a-z.]+',email)
+            password_vaildation     = re.match('^(?=.*[a-zA-Z0-9.,-]).{8,}$',password)
+            
+            if name =='' or (phone_number=='' and email=='') or username == '' or password =='':
+                return JsonResponse({'message' : 'Check your Input'}, status = 400)
+            if not email_vaildation:
+                return JsonResponse({'messsage' : 'Check your email'}, status = 400)
```

Comment:
```suggestion
            if name =='' or (phone_number=='' and email=='') or username == '' or password =='':
                return JsonResponse({'message' : 'Check your Input'}, status = 400
                
            if not email_vaildation:
                return JsonResponse({'messsage' : 'Check your email'}, status = 400)
```
if와 if 사이에 공백이 있는 것이 가독성이 좋은 것 같습니다.